libraries/chain/include/graphene/chain/account_object.hpp

```diff
@@ -96,6 +96,29 @@ namespace graphene { namespace chain {
           */
          share_type pending_vested_fees;
 
+         /**
+          * Specifies the total voting weight of this account (set in maintenance interval)
+          */
+         struct voting_stat
+         {
+            bool has_proxy;
+            uint64_t self_voting_power;
+            flat_map<account_id_type, uint64_t> proxy_for;
+
+            const uint64_t get_total_voting_power() const
+            {
+               if( !has_proxy ) {
+                  uint64_t total = self_voting_power;
+                  for( const auto& pair : proxy_for )
+                     total += pair.second;
```

Comment:
was ist pair.second?

Reply:
Das ist quasy die `voting_power`, die dieser account von anderen bekommen hat. Ich trag im maintenance immer die `account_id` vom "`sender`" ein und die "gesendete"  `voting_power`, dann sehen wir von wem der acc VP bekommen hat und wie viel.

Reply:
Joar, aber warum heißt das second?

Reply:
> Joar, aber warum heißt das second?

Es ist eine C++ Map. Die besteht aus genau zwei teilen. Dem *ersten* und dem *zweiten* :)

Reply:
Aso das ist nicht von mir. Das ist ganz normal maps. In std::map werden immer std::pairs reingelegt und die haben halt die member pair.first pair.second. Quasi unser das C++ Key Value Ding. War nicht meine Idee :D

Reply:
Danke!